packages/storybook8/stories/Concepts/Theming/Docs.mdx

```diff
@@ -78,7 +76,7 @@ But if you wish to override the mapping, we recommend that you use the `useTheme
 use the component's `styles` prop like in the example below.
 
 <Source code={ThemedButtonText} />
-<ThemedButton />
+<Canvas of={ThemingStories.ThemedButtonDocsOnly} sourceState={'none'} layout={'padded'} />
```

Comment:
Oh this is a really good find. I think we can make a similar change to a few pages (I know I made a change to Styling yesterday that can benefit from this) Does this mean we can delete the `ThemeButton.story.tsx` (and all the other applicable `*.story.tsx` in this folder) now.

Reply:
Yeah, this can be used at all the places where we are directly rendering the components without snippets

Reply:
Are there other issues on this one or can we resolve this and land this PR? @JoshuaLai